hoot-core-test/src/test/cpp/hoot/core/ops/CalculateStatsOpTest.cpp

```diff
@@ -111,10 +106,9 @@ class CalculateStatsOpTest : public CppUnit::TestFixture
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Conflatable Features"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Untagged Feature Count"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Unconflatable Features"));
-    CPPUNIT_ASSERT_EQUAL(12.0, calcStatsOp->getSingleStat("Total Features Processed By Conflation"));
     CPPUNIT_ASSERT_EQUAL(12.0, calcStatsOp->getSingleStat("Total Conflated Features"));
     CPPUNIT_ASSERT_EQUAL(19.0, calcStatsOp->getSingleStat("Total Unmatched Features"));
-    CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Features Marked for Review"));
+    CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Features Involved in a Review"));
     CPPUNIT_ASSERT_EQUAL(0.0, calcStatsOp->getSingleStat("Total Number of Reviews to be Made"));
```

Comment:
"involved in a review" is more intuitive to me, but I don't feel that strongly about it b/c I know how hoot works under the covers, so not a big deal to me...we can let the actual users decide if the name is confusing or not.  The "involved in" part was meant to better convey that a single feature may be involved in multiple reviews...but that's probably more of training issue anyway.

What I do feel strongly about is that there is no test in the UI to catch this change I made as an error.  There will be times we need to change stat names, add them, get rid of them, etc in the future, no doubt.  I battled about this a lot during the writing of pdf report generation code, and eventually convinced those involved to decouple the report pdf stat names from the c++ code to a degree and add unit tests so that we knew when something in the report broke when stats change.  So, I'll revert that change on the condition that a test is soon added that lets a person know they also need to change ui stat name text when changing c++ stat text.

A better longer term solution would be to decouple them with some kind of shared config setting...possibly managed by the service code.